cmd/logger.go

```diff
@@ -92,16 +106,21 @@ func (log *Logger) EnableQuiet() {
 	log.quiet = true
 }
 
+// EnableJSON - outputs logs in json format.
+func (log *Logger) EnableJSON() {
+	log.json = true
+}
+
 // Println - wrapper to console.Println() with quiet flag.
 func (log *Logger) Println(args ...interface{}) {
-	if !log.quiet {
+	if !log.quiet && !log.json {
```

Comment:
isn't `quiet` overriding `json`?  unable to understand why this check is needed.